packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -204,6 +204,30 @@ export class EditorPage {
 
 	//#region Basic Entry
 
+	/**
+	 * Loads HTML into the editor.
+	 * @param content Content to load.
+	 */
+	async loadHtmlContent( content: string ) {
```

Comment:
I wish we could reuse `loadBlocksFromHtml()` from Gutenberg. Perhaps we can move to the utils package and reuse from there?

Reply:
We probably can't reuse the one from Gutenberg, Calypso's editor is a bit more complex (iframed within iframe sometimes) so we can't reuse Gutenberg's one until the Calypso's iframe is completely removed.

Reply:
Let's add some TODOs here then - ideally we should be able to remove the iframe soon, and then it would be nice to reuse what we can.